src/ui/MiniChart/Area.svelte

```diff
@@ -19,12 +55,42 @@
   }
 </script>
 
-<Chart {data} {width} {height} {valueKey} class={className} {style} let:points let:linePoints>
+<Chart
+  {data}
+  {width}
+  {height}
+  {valueKey}
+  {style}
+  class="relative {className}"
+  on:mousemove={onMouseMove}
+  on:mouseleave={onMouseLeave}
+  let:points
+  let:linePoints
+>
   <polyline points={getAreaPoints(points, linePoints)} fill="url(#{id}-area)" />
   <defs>
     <linearGradient id="{id}-area" x1="0" x2="0" y1="0" y2="2">
       <stop offset="0%" stop-color="var(--color)" stop-opacity="0.3" />
       <stop offset="60%" stop-color="var(--white)" stop-opacity="0" />
     </linearGradient>
   </defs>
+
+  {#if tooltipVisible && offset}
+    <line x1={offset} x2={offset} y1="0" y2={height} stroke="var(--waterloo)" />
+
+    <foreignObject {width} {height}>
+      <tooltip class="c-fiord">{valueFormatted}</tooltip>
+    </foreignObject>
+  {/if}
 </Chart>
+
+<style>
+  tooltip {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: rgba(255, 255, 255, 0.7);
```

Comment:
This type of background will not be automatically translated to `night-mode`. Better to use `background: var(--white);` with `opacity` rule